### PR TITLE
ISPN-14682 Do not retrieve entries when no encoding in cache

### PR DESCRIPTION
**Pull request by dpanshug**
https://issues.redhat.com/browse/ISPN-14682

### CODE REVIEW

File: src/app/providers/CacheDetailProvider.tsx [hunk 1]
```diff
@@ -66,7 +66,7 @@ const CacheDetailProvider = ({ children }) => {
               })
               .finally(() => {
                 setLoading(false);
-                setLoadingEntries(true);
+                cache.queryable && setLoadingEntries(true);
```
Comment (karesti) [inline src/app/providers/CacheDetailProvider.tsx:69]: this is not ok. the cache being queriable is not the solution

**Review by karesti — CHANGES_REQUESTED** [review]

**Comment by karesti** [comment]
@dpanshug check here https://github.com/infinispan/infinispan-console/pull/338

No encoding means no entries tab at all.

There are cases that the entries fetch can fail. try creating caches with all the encodings and putting data on them

File: src/app/providers/CacheDetailProvider.tsx [hunk 2]
```diff
@@ -40,6 +40,13 @@ const CacheDetailProvider = ({ children }) => {
     }
   };
 
+  const isEncodingAvailable = (): boolean => {
```
Comment (karesti) [inline src/app/providers/CacheDetailProvider.tsx:43]: export this to the file 

the Empty encoding is an encoding type enum. use ||, not and
can be inlined in a single like

```
return cache.encoding.key  ... ||  ... 
```

**Review by karesti — CHANGES_REQUESTED** [review]

File: src/app/utils/encodingUtils.ts [hunk 3]
```diff
@@ -0,0 +1,3 @@
+export function isEncodingAvailable(cache: DetailedInfinispanCache): boolean {
+  return cache.encoding.key !== 'Empty' || cache.encoding.value !== 'Empty';
```
Comment (dpanshug) [inline src/app/utils/encodingUtils.ts:2]: @karesti let me know, if this okay, so I can update backport

Comment (karesti) [reply src/app/utils/encodingUtils.ts:2]: you can use the enum

**Review by karesti — APPROVED** [review]

**Comment by karesti** [comment]
error seems unrelated so I will merge but we need to fix this :)

**Pull request merged**